frontend/src/lib/components/PropertiesTable/PropertiesTable.tsx

```diff
@@ -136,6 +138,16 @@ function ValueDisplay({
                         valueComponent
                     )}
                     <div className="property-value-type">{propertyType || valueType}</div>
+                    {(propertyType === PropertyType.String && valueType === 'number') ||
+                    (propertyType === PropertyType.Numeric && valueType === 'string') ? (
+                        <Tooltip
+                            title={`This property's type is set to "${propertyType}", yet the displayed value is of type "${valueType}". Click to correct.`}
```

Comment:
Just usability-wise: Not everyone is clicking.

Reply:
Not sure I understand...? 

I'm not happy with the usability of this entire flow, but I hope that a message like this will just help lessen the support load a bit.

Reply:
Sorry, maybe "wording" instead of usability would have been more better. I was hinting at finding something else to say than "Click to correct." since people on e.g. an iPad are not clicking. Maybe just "Edit here." instead.

Reply:
Aha, I get it now. "Edit here" would've been better, but 🤷. The entire thing could use a change, so I'm not sure which the worst UX offence we have here 😅